src/Engines/Liquid.js

```diff
@@ -172,20 +172,57 @@ class Liquid extends TemplateEngine {
 		let tokenizer = new Tokenizer(args);
 		let parsedArgs = [];
 
-		let value = tokenizer.readValue();
+		function readValue() {
+			let value = tokenizer.readHash() ?? tokenizer.readValue();
+			// readHash() treats unmarked identifiers as hash keys with undefined
+			// values, but we want to parse them as positional arguments instead.
+			return value?.kind === 64 && value.value === undefined ? value.name : value;
```

Comment:
I’m seeing some errors here, from LiquidJS’s types: `value` and `name` do not exist on type `HashToken | ValueToken | RangeToken`